cli/internal/specs/v0/source.go

```diff
@@ -68,6 +70,10 @@ func (s *Source) SetDefaults() {
 	if s.Spec == nil {
 		s.Spec = make(map[string]any)
 	}
+	if s.SkipDependentTables == nil {
+		b := true
+		s.SkipDependentTables = &b
```

Comment:
dark magic:
```suggestion
		s.SkipDependentTables = &[]bool{true}[0]
```

Reply:
just do `lo.ToPtr(true)` without dark magic 😂

Reply:
> just do `lo.ToPtr(true)` without dark magic 😂

Yeah didn't want to add it just for this 😄